models/views/scratch/snowplow_unified_pv_scroll_depth.sql

```diff
@@ -60,6 +62,41 @@ select
   cast(round(100*(greatest(hmin, 0)/cast(doc_width as {{ type_float() }}))) as {{ type_float() }}) as relative_hmin, -- brackets matter: because hmin is of type int, we need to divide before we multiply by 100 or we risk an overflow
   cast(round(100*(least(hmax + br_viewwidth, doc_width)/cast(doc_width as {{ type_float() }}))) as {{ type_float() }}) as relative_hmax,
   cast(round(100*(greatest(vmin, 0)/cast(doc_height as {{ type_float() }}))) as {{ type_float() }}) as relative_vmin,
-  cast(round(100*(least(vmax + br_viewheight, doc_height)/cast(doc_height as {{ type_float() }}))) as {{ type_float() }}) as relative_vmax -- not zero when a user hasn't scrolled because it includes the non-zero viewheight
+  cast(round(100*(least(vmax + br_viewheight, doc_height)/cast(doc_height as {{ type_float() }}))) as {{ type_float() }}) as relative_vmax, -- not zero when a user hasn't scrolled because it includes the non-zero viewheight
+
+  cast(null as {{ type_int() }}) as last_list_item_index,
+  cast(null as {{ type_int() }}) as list_items_count,
+  cast(null as {{ type_int() }}) as list_items_percentage_scrolled
 
 from prep
+
+{% if var('snowplow__enable_screen_summary_context', false) %}
+union all
+
+select
+  t.view_id,
+  t.session_identifier,
+
+  cast(null as {{ type_int() }}) as doc_width,
+  cast(null as {{ type_int() }}) as doc_height,
+
```

Comment:
Could we not use the content width/height here? I understand this comes from the web and the other is from a mobile context, but it would help users keep a unified field approach?

Reply:
Good idea, have used the content width and height from screen summary here.